src/dag/transitive.d.ts

```diff
@@ -0,0 +1,29 @@
+import Graph from 'graphology';
+import {Attributes} from 'graphology-types';
+
+export type TransitiveIterationCallback<
+  NodeAttributes extends Attributes = Attributes,
+  EdgeAttributes extends Attributes = Attributes
+> = (
+  inEdge: string,
+  outEdge: string,
+  inEdgeAttributes: EdgeAttributes,
+  outEdgeAttributes: EdgeAttributes,
+  source: string,
+  node: string,
+  target: string,
+  sourceAttributes: NodeAttributes,
+  nodeAttributes: NodeAttributes,
+  targetAttributes: NodeAttributes
+) => void;
+
+export function forEachTransitiveRelation(
```

Comment:
This function is lacking its generic arguments no? I don't think `callback: TransitiveIterationCallback` alone without the generics will do more than inferring unkown node & edge attributes types.